src/pandroid/app/src/main/java/com/panda3ds/pandroid/utils/GameUtils.java

```diff
@@ -0,0 +1,64 @@
+package com.panda3ds.pandroid.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.net.Uri;
+
+import com.google.gson.Gson;
+import com.panda3ds.pandroid.app.GameActivity;
+import com.panda3ds.pandroid.app.PandroidApplication;
+import com.panda3ds.pandroid.data.game.GameMetadata;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class GameUtils {
+    private static final String KEY_GAME_LIST = "gameList";
+    private static final ArrayList<GameMetadata> games = new ArrayList<>();
+    private static SharedPreferences data;
+    private static final Gson gson = new Gson();
+
+    public static void initialize() {
+        data = PandroidApplication.getAppContext().getSharedPreferences(Constants.PREF_GAME_UTILS, Context.MODE_PRIVATE);
+
+        GameMetadata[] list = gson.fromJson(data.getString(KEY_GAME_LIST, "[]"), GameMetadata[].class);
+        games.clear();
```

Comment:
Can't you do something like `games = Arrays.asList(list)`? And if not, isn't `games` always clear during initialize?

Reply:
Arrays.asList return an unmodifiable Array